bodhi/server/config.py

```diff
@@ -457,8 +485,14 @@ class BodhiConfig(dict):
         'openid_template': {
             'value': '{username}.id.fedoraproject.org',
             'validator': str},
-        'pagure_flatpak_namespace': {
-            'value': 'modules',
+        'pagure_namespaces': {
+            'value': ('rpm:rpms, module:modules, container:container, flatpak:modules'),
```

Comment:
``flatpak:modules``?

Reply:
Yep, it is currently set as `modules` by default (see the two removed lines above), then redefined to `flatpak` in ansible config template.
I think we can now make it to be `flatpak` by default, I'm going to change this line.